### ISSUE TITLE
Problems running guis through yarprun

### ISSUE THREAD

**Issue by lornat75** [robotology/icub-main#272]
When yarprun is managed through icub-cluster, executing guis with yarpmanager requires setting display property to true for the corresponding node:

    <node display=":0.0">nodename</node> 

in the cluster-config.xml file.

Long explanation:

yarpmanager executes modules through yarprun, yarprun can spawn gui processes only if has been granted access to the display. This happens if yarprun is executed from an xterm, but requires extra commands if yarprun is executed via ssh. icub-cluster can be configured correctly by setting the display option to true.

In short the script runs these commands prior to execute yarprun:
   export DISPLAY=:0.0
   xhost +

To set a different value of DISPLAY use:

     <node display=":x.y">nodename</node>
